fastq_filtrator.py

```diff
@@ -0,0 +1,148 @@
+def main():  # главная программа
+    input_fastq = input("Input the fastq file name ->")
+    output_file_prefix = input("Input the output file prefix ->")
+    yfilename = output_file_prefix + '_passed.fastq'
+    zfilename = output_file_prefix + '_failed.fastq'
+    leng_min = 0
+    leng_max = 2**32
+    gc_bounds_min = 0
+    gc_bounds_max = 100
+    quality_threshold = 0
+    save_filtered = False
```

Comment:
Подразумевалось, что значения по-умолчанию должны устанавливаться при определении функции. Например, вот так:
```suggestion
def main(input_fastq, output_file_prefix, gc_bounds=(0, 100), length_bounds=(0, 2**32), quality_threshold=0, save_filtered=False):  # главная программа
    yfilename = output_file_prefix + '_passed.fastq'
    zfilename = output_file_prefix + '_failed.fastq'
```